dashboards-observability/server/common/metrics/metrics_helper.ts

```diff
@@ -64,8 +64,10 @@ export function addRequestToMetric(
   const timeKey = getKey(Date.now());
   const rollingCounter = time2CountWin.get(timeKey) || _.cloneDeep(DEFAULT_ROLLING_COUNTER);
 
+  // @ts-ignore not sure why 'request' can't be indexed
   rollingCounter[component][request][counter]!++;
```

Comment:
The error is ` Type 'RequestType<T>' cannot be used to index type 'CounterType[T]'.`, let me know if anyone knows how I should change it and i can remove the `@ts-ignore`